src/backend/TrafficCourts/Citizen.Service/Mappings/MappingProfile.cs

```diff
@@ -1,38 +1,38 @@
-﻿using AutoMapper;
-using TrafficCourts.Citizen.Service.Models.Disputes;
-using TrafficCourts.Messaging.MessageContracts;
-
-namespace TrafficCourts.Citizen.Service.Mappings;
-
-public class NoticeOfDisputeToMessageContractMappingProfile : Profile
-{
-    public NoticeOfDisputeToMessageContractMappingProfile()
-    {
-        CreateMap<NoticeOfDispute, SubmitNoticeOfDispute>()
-            .ForMember(dest => dest.DisputeCounts, opt => opt.MapFrom(src => src.DisputeCounts));
-        CreateMap<Models.Disputes.DisputeCount, Messaging.MessageContracts.DisputeCount>();
-
-        CreateMap<SubmitNoticeOfDispute, NoticeOfDispute>()
-            .ForMember(dest => dest.DisputeCounts, opt => opt.MapFrom(src => src.DisputeCounts))
-            .ForMember(dest => dest.ViolationTicket, opt => opt.MapFrom(src => src.ViolationTicket));
+﻿using AutoMapper;
+using TrafficCourts.Citizen.Service.Models.Disputes;
+using TrafficCourts.Messaging.MessageContracts;
+
+namespace TrafficCourts.Citizen.Service.Mappings;
+
+public class NoticeOfDisputeToMessageContractMappingProfile : Profile
+{
+    public NoticeOfDisputeToMessageContractMappingProfile()
+    {
+        CreateMap<NoticeOfDispute, SubmitNoticeOfDispute>()
+            .ForMember(dest => dest.DisputeCounts, opt => opt.MapFrom(src => src.DisputeCounts));
+        CreateMap<Models.Disputes.DisputeCount, Messaging.MessageContracts.DisputeCount>();
+
+        CreateMap<SubmitNoticeOfDispute, NoticeOfDispute>()
+            .ForMember(dest => dest.DisputeCounts, opt => opt.MapFrom(src => src.DisputeCounts))

```

Comment:
If the members property names match, I am pretty sure they get auto mapped. You dont need to explicitly define this. this is the point of automapper.

Reply:
checking it out with this line removed stay tuned...

Reply:
and you are of course correct its not needed